### ISSUE TITLE
Add ability to enable or disable IR emitter

### ISSUE THREAD

**Issue by SirDifferential** [orbbec/astra#25]
Currently it's not possible to toggle the IR emitter via the astra API or seemingly via the OpenNI setParameter() calls. In order to properly use both the IR and depth images, this feature is needed. So far we've been doing this by putting a blanket on the IR emitter :)

**Comment by JoshBlake** [comment]
I agree, this should be integrated into Astra SDK. It's on our list of things to do, but thanks for sharing your interest in this feature! If you are in urgent need for this, we can give you some sample code if you email us at info@orbbec3d.com and request it.
